components/web_discovery/common/BUILD.gn

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/components/web_discovery/buildflags/buildflags.gni")
+
+assert(enable_web_discovery_native)
+
+component("common") {
+  output_name = "web_discovery_common"
```

Comment:
why are you changing the output name here? There's also no reason for this to be in `common` or really to have process directories at all for this component